TestingLowerBounds/Testing/Binary.lean

```diff
@@ -250,38 +250,154 @@ lemma bayesBinaryRisk_eq (μ ν : Measure 𝒳) (π : Measure Bool) :
   rw [bayesianRisk, lintegral_fintype, mul_comm (π {false}), mul_comm (π {true})]
   simp
 
+variable {π : Measure Bool}
+
+--rename this and put it in a better place
+lemma mem_set_bool (s : Set Bool) : s = ∅ ∨ s = {true} ∨ s = {false} ∨ s = {true, false} := by
```

Comment:
The `mem` part of the name is strange, since this is not about membership. `cases_set_bool` perhaps?
```suggestion
lemma cases_set_bool (s : Set Bool) : s = ∅ ∨ s = {true} ∨ s = {false} ∨ s = {true, false} := by
```